<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Owner_PlaceCargo</class>
 <widget class="QWidget" name="Owner_PlaceCargo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create New Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Departure Point</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Departure_combo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string comment="1">North UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="2">East UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="3">West UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="4">South UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Delivery Point</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Delivery_combo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>North UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>East UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>West UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>South UK</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="cargo_weight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight of your Cargo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Conditions of the storing</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Condtitions_combo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fridge(frozen goods)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fragile Cargo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Liquid goods</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chemicals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Place_cargo_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate_price">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate The
 Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type in the number in KG</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>The maximum weight is 5000kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estimated shipping cost:</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_price_text">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cargo_note">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>481</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Note(you can explain what is inside your cargo or sprecify conditions)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
